doc/release-notes-1.3.0.rst

```diff
@@ -0,0 +1,571 @@
+* :ref:`ug_nrf5340` - updated
+* :ref:`ug_nrf52` - added
+* :ref:`ug_thingy91` - added :ref:`thingy91_serialports`
+* :ref:`ug_nfc` - added
+* :ref:`ug_bootloader` - added :ref:`ug_bootloader_adding`
+* :ref:`cloud_client` - updated
+* :ref:`crypto_test` - added
+* :ref:`libraries` - improved the structure of the library documentation
+* :ref:`bt_mesh` (and subpages) - added
+* :ref:`nrf_bt_scan_readme` - updated
+* :ref:`at_cmd_readme` - added
+* :ref:`coap_utils_readme` - added
+* :ref:`tnep_poller_readme` and :ref:`tnep_tag_readme` - updated
+* :ref:`nrf_desktop_config_channel_script` - updated
+* :ref:`nrfxlib:bsdlib` - added documentation about :ref:`nrfxlib:gnss_extension`
+* :ref:`nrfxlib:mpsl` - added documentation about :ref:`nrfxlib:mpsl_timeslot`, :ref:`nrfxlib:mpsl_radio_notification`, and :ref:`nrfxlib:mpsl_tx_power_control`
+* :ref:`nrfxlib:nfc` - added documentation about :ref:`nrfxlib:type_2_tag` and :ref:`nrfxlib:type_4_tag`, updated the :ref:`nrfxlib:nfc_integration_notes`
+* :ref:`nrfxlib:nrf_security_readme` - updated
+
+Known issues
+************
+
+nRF9160
+=======
+
+* The :ref:`asset_tracker` applications prints warnings and error messages during successful FOTA. (NCSDK-5574)
+* The :ref:`lte_sensor_gateway` sample crashes when Thingy:52 is flipped. (NCSDK-5666)
+
+From v1.2.0
+-----------
+
+* The :cpp:func:`nrf_send` function in the :ref:`nrfxlib:bsdlib` might be blocking for several minutes, even if the socket is configured for non-blocking operation.
+  The behavior depends on the cellular network connection.
+* The :ref:`asset_tracker` sample might show up to 2.5 mA current consumption in idle mode with ``CONFIG_POWER_OPTIMIZATION_ENABLE=y``.
+* The SEGGER Control Block cannot be found by automatic search by the RTT Viewer/Logger.
+  As a workaround, set the RTT Control Block address to 0 and it will try to search from address 0 and upwards.
+  If this does not work, look in the ``builddir/zephyr/zephyr.map`` file to find the address of the ``_SEGGER_RTT`` symbol in the map file and use that as input to the viewer/logger.
+* nRF91 fails to receive large packets (over 4000 bytes).
+* nrf_connect fails if called immediately after initialization of the device.
+  A delay of 1000 ms is required for this to work as intended.
+
+nRF5
+====
+
+nRF5340
+-------
+
+* FOTA with the :ref:`zephyr:smp_svr_sample` does not work.
+
+Multi-Protocol Service Layer (MPSL)
+-----------------------------------
+
+* The Antenna Diversity feature is not supported on nRF52840 devices. (KRKNWK-6361)
+
+Thread
+------
+
+* It is not possible to build Thread samples using SEGGER Embedded Studio (SES).
+  SES does not support .cpp files in |NCS| projects. (NCSDK-5014)
+* It is not possible to provision the :ref:`coap_client_sample` sample to servers that it cannot directly communicate with.
+  This is because Link Local Address is used for communication. (KRKNWK-6358) (?)
```

Comment:
not sure what this question mark is for. If it is about issue being closed here is the still open issue: KRKNWK-6427

Reply:
This was the original input:

> It is not possible to provision CoAP Client with servers it is not able to communicate directly as Link Local Address is used for communication (KRKNWK-6358).

I wasn't sure if my interpretation of what communicates with what here is correct. ;)

Reply:
ok. Looks good.